src/Tizen.NUI/src/public/WindowEvent.cs

```diff
@@ -920,6 +977,14 @@ public EffectTypes Type
             }
         }
 
+        /// <summary>
+        /// KeyboardRepeatSettingsChangedArgs
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public class KeyboardRepeatSettingsChangedArgs : EventArgs
```

Comment:
I think this KeyboardRepeatSettingsChangedArgs needs be removed, because it doesn't have any info in it.
how do you think about?

Reply:
Yes, it can be removed because there's no argument and no return value here.